lib/bamboo/adapters/smtp_adapter.ex

```diff
@@ -161,6 +168,45 @@ defmodule Bamboo.SMTPAdapter do
     |> add_smtp_line(text_body)
   end
 
+  defp add_multipart_mixed_split(body, delimiter), do: "#{body}\r\n--#{delimiter}\r\n"
```

Comment:
1. Do we need an extra blank line between the body and the delimiter? (`body` already ends with an `\r\n`)
2. What do you think of using the `add_smtp_line` function instead of doing the concatenation yourself?

```elixir
defp add_multipart_mixed_split(body, delimiter) do
  add_smtp_line(body, "-#{delimiter}")
end
```